openbsd/OpenBSDMachine.c

```diff
@@ -92,11 +92,12 @@ Machine* Machine_new(UsersTable* usersTable, uid_t userId) {
    char errbuf[_POSIX2_LINE_MAX];
 
    OpenBSDMachine* this = xCalloc(1, sizeof(OpenBSDMachine));
-   Machine* super = &this->super;
+   Machine* super = (Machine*)&this->super;
```

Comment:
What's the error message for this? `this->super` should be of type `Machine` in `openbsd/OpenBSDMachine.h`.